lib/order.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :dishes, :order
+
+  def initialize
+    @dishes = { hotdog: 5.00, fries: 3.50 }
+    @order = {}
+  end
+
+  def add(dish, quantity)
+    fail "#{dish.capitalize} is not on the menu!" unless has_dish?(dish)
+    order[dish] = quantity
+  end
+
+  def total
+    item_totals.inject(:+)
+  end
+
+  def price(dish)
+    @dishes[dish]
+  end
+
+  def item_totals
```

Comment:
`item_total` seems more closely aligned with the `Order` class's  responsibilities. i.e This method should be moved to the Order class. Perhaps also rename to `order_total`?